classes/View.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class View
+{
+    private $template;
+    
+    public function __construct($template)
+    {
+        $this->template = $template;
+    }
+    
+    public function render($params = array())
+    {
+        // Affichage de la topbar
+        $topbar="views/topbar_frontend.php";
+        
+        extract($params); // extrac() parcourt le tableau de $params et créé posts, responses, post, etc.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function extract() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)